### PR TITLE
[UI] Playground modal to Deploy Meshery locally | CU-333dqt2

### PR DESCRIPTION
**Pull request by Chadha93**
Signed-off-by: Gaurav Chadha <gauravchadha1676@gmail.com>

*Description*
Playground modal to Deploy Meshery locally

**Notes for Reviewers**
Verbiage and content can be improved. 
AccessModal

This PR fixes #CU-333dqt2

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
Task linked: [CU-333dqt2 [Feedback] Dec 13th MeshMap - Playground launch requirements/items left.](https://app.clickup.com/t/333dqt2)

File: ui/pages/extension/AccessMesheryModal.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        <Typography>Access is limited in Meshery <span style={{ color : "#F0A303" }}>Playground.</span> <br />
+                    To get full access, Deploy Meshery locally.</Typography>
+      </div>
+      <Typography style={{ textAlign : "center", marginBottom : "1rem" }}>&#x26A1; Deploy Meshery in seconds</Typography>
+      <div style={{ display : "flex", justifyContent : "center" }}>
+        <Grid item xs={3}>
+          <Button fullWidth variant="outlined" color="primary" onClick={(e) => handlePage(e)}>Get Started</Button>
+        </Grid>
+      </div>
+    </div>
```
Comment (leecalcote) [inline ui/pages/extension/AccessMesheryModal.js:103]:
```suggestion
      <div className={classes.headerWrapper}>
        <Typography className={classes.header} variant="h6">The Cloud Native Playground</Typography>
        <div style={{ display : "flex", justifyContent : "flex-end", whiteSpace : "nowrap", position : "relative" }}>
          <IconButton key="close" aria-label="Close" color="inherit" onClick={closeForm}>
            <CloseIcon className={classes.closing} />
          </IconButton>
        </div>
      </div>
      <div className={classes.imgWrapper}>
        <img width="60%" height="60%" src="/static/img/meshery-logo-light-text-side.png" />
      </div>
      <div style={{ display : "flex", justifyContent : "center", padding : "1.5rem", width : "100%" }} >
        <Typography>Meshery Playground gives you hands-on experience using not just Kubernetes, but every CNCF project and surrounding ecosystem. Choose your <a href="https://layer5.io/learn/learning-paths">Learning Path</a> and follow their labs in your browser as you visually and collaboratively learn by doing without having to install a single thing.</Typography>
      </div>
      <Typography style={{ textAlign : "center", marginBottom : "1rem" }}>To ensure that Meshery Playground remains a clean sandbox of all to use, many of Meshery's features are disabled. For full access to all of Meshery's features, deploy your own instance of Meshery.</Typography>
      <div style={{ display : "flex", justifyContent : "center" }}>
        <Grid item xs={3}>
          <Button fullWidth variant="outlined" color="primary" onClick={(e) => handlePage(e)}>Get Started</Button>
        </Grid>
      </div>
    </div>
```

File: ui/pages/extension/AccessMesheryModal.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
+import React, {
+} from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography, Grid, Button, IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+
+const styles = makeStyles((theme) => ({
+  paper : {
+    position : 'absolute',
```
Comment (leecalcote) [inline ui/pages/extension/AccessMesheryModal.js:9]: use flex, please.

File: ui/pages/extension/AccessMesheryModal.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    width : "30rem",
+    height : "20rem",
```
Comment (leecalcote) [inline ui/pages/extension/AccessMesheryModal.js:11]: move away from static sizing where you can. use max-height, please with overflow set to scroll.

Screenshot 2022-12-16 at 4 28 13 PM

File: ui/pages/extension/AccessMesheryModal.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    backgroundColor : theme.palette.background.paper,
+    boxShadow : theme.shadows[5],
+    borderRadius : "4px",
```
Comment (leecalcote) [inline ui/pages/extension/AccessMesheryModal.js:14]: Use `rem`. Why is the styling not part of modal component?

File: ui/pages/extension/AccessMesheryModal.js [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    top : "0",
+    bottom : "0",
+    left : "0",
+    right : "0",
+    margin : "auto",
```
Comment (leecalcote) [inline ui/pages/extension/AccessMesheryModal.js:19]: use flex, please.

File: ui/pages/extension/AccessMesheryModal.js [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+      fontSize : "1rem"
+    },
+  },
+  imgWrapper : {
+    display : "flex",
+    justifyContent : "center"
+  },
+  headerWrapper : {
+    marginBottom : 12,
+    borderRadius : "4px 4px 0 0",
```
Comment (leecalcote) [inline ui/pages/extension/AccessMesheryModal.js:42]: Don't use static sizing.

File: ui/pages/extension/AccessMesheryModal.js [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+  },
+  header : {
+    display : "flex",
+    justifyContent : "center",
+    position : "absolute",
+    width : "100%",
+    top : "0.7rem",
+    fontWeight : "bold",
+    ["@media (max-width: 455px)"] : {
+      fontSize : "1rem"
```
Comment (leecalcote) [inline ui/pages/extension/AccessMesheryModal.js:33]: Use calc() or vw or something else, if you can. Try to be rid of any media queries.

File: ui/pages/extension/AccessMesheryModal.js (hunk 5 above)
Comment (leecalcote) [reply ui/pages/extension/AccessMesheryModal.js:19]: This media query doesn't seem to be having an effect, nor should we need it if you use less fragile CSS.

Screenshot 2022-12-16 at 4 31 45 PM

File: ui/pages/extension/AccessMesheryModal.js [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+    margin : "auto",
+    ["@media (max-width: 455px)"] : {
+      width : "100%"
+    },
+    zIndex : 5
+  },
+  header : {
+    display : "flex",
+    justifyContent : "center",
+    position : "absolute",
```
Comment (leecalcote) [inline ui/pages/extension/AccessMesheryModal.js:28]: No, sir.

File: ui/pages/extension/AccessMesheryModal.js [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+    color : '#fff',
+    backgroundColor : "#252E31",
+    fontSize : "1rem",
+  },
+  closing : {
+    transform : "rotate(-90deg)",
+    "&:hover" : {
+      transform : "rotate(90deg)",
+      transition : "all .3s ease-in",
+      cursor : "pointer",
```
Comment (leecalcote) [inline ui/pages/extension/AccessMesheryModal.js:55]: Why is this not a standard part of the modal component?

File: ui/pages/extension/AccessMesheryModal.js [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+          </IconButton>
+        </div>
+      </div>
+      <div className={classes.imgWrapper}>
+        <img width="60%" height="60%" src="/static/img/meshery-logo-light-text-side.png" />
+      </div>
+      <div style={{ display : "flex", justifyContent : "center", padding : "1.5rem", width : "100%" }} >
+        <Typography>Meshery Playground gives you hands-on experience using not just Kubernetes, but every CNCF project and surrounding ecosystem. Choose your <a href="https://layer5.io/learn/learning-paths">Learning Path</a> and follow their labs in your browser as you visually and collaboratively learn by doing without having to install a single thing.</Typography>
+      </div>
+      <Typography style={{ textAlign : "center", marginBottom : "1rem" }}>To ensure that Meshery Playground remains a clean sandbox of all to use, many of Meshery's features are disabled. For full access to all of Meshery's features, deploy your own instance of Meshery.</Typography>
```
Comment (leecalcote) [inline ui/pages/extension/AccessMesheryModal.js:96]:
```suggestion
      <Typography style={{ textAlign : "center", marginBottom : "1rem" }}>To ensure that Meshery Playground remains a clean sandbox of all to use, many of Meshery&apos;s features are disabled. For full access to all of Meshery&apos;s features, deploy your own instance of Meshery.</Typography>
```

File: ui/pages/extension/AccessMesheryModal.js (hunk 3 above)
Comment (Chadha93) [reply ui/pages/extension/AccessMesheryModal.js]: Done

File: ui/pages/extension/AccessMesheryModal.js (hunk 4 above)
Comment (Chadha93) [reply ui/pages/extension/AccessMesheryModal.js]: Fixed, now, it is.

File: ui/pages/extension/AccessMesheryModal.js (hunk 9 above)
Comment (Chadha93) [reply ui/pages/extension/AccessMesheryModal.js]: Now it is.

File: ui/pages/extension/AccessMesheryModal.js (hunk 8 above)
Comment (Chadha93) [reply ui/pages/extension/AccessMesheryModal.js:13]: It was required here. Also used by MUI dialogs/modals.

**Comment by Chadha93** [comment]
@leecalcote Updated video 

https://user-images.githubusercontent.com/65453826/208235515-489f6322-81de-44c5-95cd-91f396fd1b90.mp4

File: ui/pages/extension/AccessMesheryModal.js (hunk 8 above)
Comment (leecalcote) [reply ui/pages/extension/AccessMesheryModal.js:13]: "used by MUI dialogs/modals." - yes
"It was required here." - no, this is a choice. Other positioning CAN be used.

**Comment by leecalcote** [comment]
Thank you for the updated video. This helps.

File: ui/pages/extension/AccessMesheryModal.js [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+          <div className={classes.imgWrapper}>
+            <img width="60%" height="60%" src="/static/img/meshery-logo-light-text-side.png" />
+          </div>
+          <Typography gutterBottom>
+            Meshery Playground gives you hands-on experience using not just Kubernetes, but every CNCF project and surrounding ecosystem.
+            Choose your <a href="https://layer5.io/learn/learning-paths">Learning Path</a> and follow their labs in your browser as you visually and collaboratively learn by doing without having to install a single thing.
+          </Typography>
+          <Typography gutterBottom>
+            To ensure that Meshery Playground remains a clean sandbox of all to use, many of Meshery&apos;s features are disabled.
+            aaFor full access to all of Meshery&apos;s features, deploy your own instance of Meshery.
```
Comment (leecalcote) [inline ui/pages/extension/AccessMesheryModal.js:66]:
```suggestion
            For full access to all of Meshery&apos;s features, deploy your own instance of Meshery.
```

File: ui/pages/extension/AccessMesheryModal.js [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+            <CloseIcon />
+          </IconButton>
+        </DialogTitle>
+        <DialogContent>
+          <div className={classes.imgWrapper}>
+            <img width="60%" height="60%" src="/static/img/meshery-logo-light-text-side.png" />
+          </div>
+          <Typography gutterBottom>
+            Meshery Playground gives you hands-on experience using not just Kubernetes, but every CNCF project and surrounding ecosystem.
+            Choose your <a href="https://layer5.io/learn/learning-paths">Learning Path</a> and follow their labs in your browser as you visually and collaboratively learn by doing without having to install a single thing.
```
Comment (leecalcote) [inline ui/pages/extension/AccessMesheryModal.js:62]:
```suggestion
            Meshery Playground gives you hands-on experience with designing cloud native systems - from your browser - using every CNCF project.
            Choose a <a href="https://layer5.io/learn/learning-paths">Learning Path</a> and work through labs as you visually and collaboratively learn-by-doing without having to install a single thing.
```

**Review by leecalcote — DISMISSED** [review]

**Pull request merged**